tmva/tmva/test/DNN/CNN/CMakeLists.txt

```diff
@@ -8,8 +8,15 @@ find_package(ROOT REQUIRED)
 
 set(Libraries Core MathCore Matrix TMVA)
 
+
 if (CUDA_FOUND AND tmva-gpu)
-include_directories(${ROOT_INCLUDE_DIRS} ${CUDA_INCLUDE_DIRS})
+    SET(DNN_CUDA_LIBRARIES dnn_cuda ${CUDA_CUBLAS_LIBRARIES})
+
+    CUDA_ADD_EXECUTABLE(testReshapeCuda TestReshapeCuda.cxx)
+    TARGET_LINK_LIBRARIES(testReshapeCuda ${Libraries} ${DNN_CUDA_LIBRARIES})
+    ROOT_ADD_TEST(TMVA-DNN-CNN-ReshapeCuda COMMAND testReshapeCuda)
+    include_directories(${ROOT_INCLUDE_DIRS} ${CUDA_INCLUDE_DIRS})
```

Comment:
Please use a consistent style with the rest of the code, that is, command not from ROOT are in lower case, commands from ROOT are in upper case.

Reply:
Hello @amadio and thanks for the review. I am not very experience with `cmake`, so I tried to follow the naming convention in the `tmva` module, for example `test/DNN/CMakeLists.txt`. If I understand correctly, the command `target_link_libraries` must be lowercase. Should I also make this change to the `test/DNN/CMakeLists.txt` for consistency even though my changes do not touch the file otherwise?

Reply:
It's ok to only make the changes in the files you are adding. And you're right, not all files follow the convention already. TMVA was a separate project before, so sometimes it has different conventions than the rest of ROOT.

Reply:
Great, thanks :)

Reply:
Looks good now, thanks.